compositor_render/src/renderer.rs

```diff
@@ -82,14 +86,16 @@ pub enum RendererRegisterTransformationError {
 
 impl Renderer {
     pub fn new(init_web: bool) -> Result<Self, RendererNewError> {
+        let wgpu_ctx = Arc::new(WgpuCtx::new()?);
         Ok(Self {
-            wgpu_ctx: Arc::new(WgpuCtx::new()?),
+            wgpu_ctx: wgpu_ctx.clone(),
             text_renderer_ctx: TextRendererCtx::new(),
             electron_instance: Arc::new(ElectronInstance::new(9002, init_web)?), // TODO: make it configurable
             scene: Scene::empty(),
             web_renderers: TransformationRegistry::new(RegistryType::WebRenderer),
             shader_transforms: TransformationRegistry::new(RegistryType::Shader),
             image_registry: TransformationRegistry::new(RegistryType::Image),
+            common_transformations_registry: CommonTransformationsRegistry::new(&wgpu_ctx),
```

Comment:
wouldn't call it registry this, this is a static structure so it's not really correct